### PR TITLE
Update versions + minor changes

### PR DESCRIPTION
**Pull request by eric-unc**
Most notably opencv 0.66 -> 0.76

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -13,17 +13,17 @@ categories = ["multimedia::encoding", "science"]
 exclude = ["dataset", "**/*.gif"]
 
 [dependencies]
-opencv = "0.66.0"
+opencv = "0.76.4"
 clap = { version = "4.0.17", features = ["derive"] }
 rayon = "1.5"
 toml = "0.5.8"
-aedat = "1.3.1"
+aedat = { git = "https://github.com/eric-unc/aedat-rs.git", version = "1.3.2"}
```
Comment (ac-freeman) [inline Cargo.toml:20]: Don't use local references

File: src/util/reconstructor.rs [hunk 2]
```diff
@@ -90,6 +90,7 @@ impl Reconstructor {
             "file" => {
                 Decoder::new_from_file(Path::new(&(directory.clone() + "/" + &aedat_filename_0)))?
             }
+            #[cfg(target_family = "unix")]
```
Comment (ac-freeman) [inline src/util/reconstructor.rs:93]: Set up a GitHub Actions pipeline on your fork to test it for Linux & Windows before we merge. Use [this](https://github.com/ac-freeman/adder-codec-rs/blob/main/.github/workflows/rust_action_test_and_docs.yml) for inspiration.

**Review by ac-freeman — CHANGES_REQUESTED** [review]

File: .github/workflows/rust.yml [hunk 3]
```diff
@@ -6,29 +6,36 @@ on:
...
-          - os: ubuntu-latest
-            target: x86_64-unknown-linux-gnu
-            suffix: ''
-          - os: windows-latest
-            target: x86_64-pc-windows-msvc
-            suffix: .exe
-
+  check_project:
+    name: Rust project
+    runs-on: ubuntu-20.04
```
Comment (ac-freeman) [inline .github/workflows/rust.yml:12]: ? You need to build for the whole matrix

**Review by ac-freeman — CHANGES_REQUESTED** [review]

**Comment by eric-unc** [comment]
CI support is there for both Ubuntu Windows, inspired by a mix of adder-codec-rs's CI and opencv-rust's. It's on the messy side but it works. All that's left is to update the aedat reference once that's released.

**Comment by ac-freeman** [comment]
Subsumed by #22

**Pull request closed without merging**